### PR TITLE
edit

### PR DESCRIPTION
**Pull request by tjdvlf2880**
haha

### CODE REVIEW

**Review by HongSeongPil — CHANGES_REQUESTED** [review]
OK YOU DID GOOD BUT WHY

**Comment by tjdvlf2880** [comment]
> OK YOU DID GOOD BUT WHY
BECAISE ~~

**Comment by tjdvlf2880** [comment]
FUCK YTOU MA

**Review by tjdvlf2880 — COMMENTED** [review]
SHUT UP

**Comment by tjdvlf2880** [comment]
HOW

**Review by HongSeongPil — COMMENTED** [review]
WELL DONE

**Comment by HongSeongPil** [comment]
이것은 전체 메세지

File: hello.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
 test for tjdvlf2
-in slave branch
\ No newline at end of file
+in slave branch2
```
Comment (HongSeongPil) [inline hello.html:2]: 응애 메세지

Comment (HongSeongPil) [reply hello.html:2]: ㅛㄷㅁㅎ

**Review by HongSeongPil — COMMENTED** [review]
좀ㅅ 솓 려차

File: hello.html (hunk 1 above)
Comment (tjdvlf2880) [reply hello.html:2]: 녀차

**Review by HongSeongPil — APPROVED** [review]

**Pull request merged**